mlrun/db/httpdb.py

```diff
@@ -950,13 +950,17 @@ def delete_runtime_resources(
         :param force: Force deletion - delete the runtime resource even if it's not in terminal state or if the grace
             period didn't pass.
         :param grace_period: Grace period given to the runtime resource before they are actually removed, counted from
-            the moment they moved to terminal state.
+            the moment they moved to terminal state (defaults to mlrun.mlconf.runtime_resources_deletion_grace_period).
```

Comment:
this is not accurate as well, as the `runtime_resources_deletion_grace_period` is managed by the API and not by the client.

Reply:
maybe needs to be passed through the client-spec

Reply:
it is enriched on the server side only on the periodic cleanup